.github/CODEOWNERS

```diff
@@ -5,28 +5,18 @@
 
 # These are the default owners for the code and will
 # be requested for review when someone opens a pull request.
+# order is alphabetical for easier maintenance.
 #
 # Aaron Tye (atye)
-# Alexander Hoppe (hoppea2)
-# Alik Saring (alikdell)
-# AnandatDell (anandrajak1)
-# Bowen Jiang (bjiang27)
+# Aly Nathoo (anathoodell)
 # Chiman Jain (chimanjain)
-# Don Khan (donatwork)
 # Evgeny Uglov (EvgenyUglov)
-# Florian Coulombel (coulof)
 # Harish H  (HarishH-DELL)
-# Harsha Yalamanchili (harshaatdell)
 # Nitesh Rewatkar (nitesh3108)
+# Oleksandr babiychuk (alexemc)
```

Comment:
Last name should start with a capital letter.

Reply:
Fixed.